include/cliffordsynthesis/Results.hpp

```diff
@@ -0,0 +1,157 @@
+/*
+* This file is part of the MQT QMAP library which is released under the MIT license.
+* See file README.md or go to https://www.cda.cit.tum.de/research/ibm_qx_mapping/ for more information.
+*/
+#pragma once
```

Comment:
Since you use `#pragma once` here in one of the new files: try to stay consistent and use it everywhere instead of header guards.